smac/facade/smac_ac_facade.py

```diff
@@ -558,8 +554,8 @@ def __init__(self,
             'scale_perc': 5,
         }
 
+        # TODO: consider other sorts of multi-objective algorithms
```

Comment:
is this TODO up to date? by allowing arbitrary AggregationStrategy we can have other MO algorithms?

Reply:
leave, is up to date